node/mdSchema.ts

```diff
@@ -117,6 +122,14 @@ export const schemas = [
           type: 'array',
           title: 'Price Tables',
         },
+        organizationCustomFields: {
+          type: 'array',
+          title: 'Custom Fields',
+        },
+        costCenterCustomFields: {
```

Comment:
We just need one `customFields` field here too

Reply:
✅